src/index.ts

```diff
@@ -101,22 +112,23 @@ if (currentScript) {
     bufferMaxSpans,
     bufferTimeout,
     ignoreUrls,
+    propagateTraceHeaderCorsUrls,
   } = currentScript.dataset;
+
   if (!collectionSourceUrl) {
     throw new Error(
-      "data-collection-source-url needs to be defined to properly set up SumoLogic JS auto-instrumentation"
+      "data-collection-source-url needs to be defined to initialize Sumo Logic OpenTelemetry auto-instrumentation for JavaScript"
```

Comment:
maybe you can use a single source of truth when it comes to package full name ("Sumo Logic OpenTelemetry auto-instrumentation for JavaScript")

Reply:
Do you mean to move this phrase ("Sumo Logic OpenTelemetry auto-instrumentation for JavaScript") into a variable?

Reply:
I meant variable / some config / package.json (if possible). Anything that would be defined once and could be reused anytime it's needed. It's an optional suggestion and obviously does not have to be part of this PR.

Reply:
I would like to leave it as it is for now and reevaluate it later when more functionalities come.